src/libraries/System.Security.Cryptography.Encoding/src/System.Security.Cryptography.Encoding.csproj

```diff
@@ -143,7 +143,7 @@
     <Reference Include="System.Security.Cryptography.Primitives" />
     <Reference Include="System.Text.Encoding.Extensions" />
   </ItemGroup>
-  <ItemGroup Condition=" '$(TargetsOSX)' == 'true' ">
+  <ItemGroup Condition=" '$(TargetsOSX)' == 'true' or '$(TargetsiOS)' == 'true' ">
```

Comment:
If we're going to have a lot of things that say OSX/iOS are the same, maybe we want to make TargetsDarwin, which covers macOS, iOS, iPadOS, tvOS, watchOS, anySimilarFutureOS; and have the build hierarchy be

* AnyOS
* Unix : AnyOS
* Windows_NT : AnyOS
* Linux : Unix
* FreeBSD : Unix
* Darwin : Unix
* OSX : Darwin
* iOS : Darwin
* ...

(A rebranding cleanup of OSX to macOS now that there aren't any remaining supported "OSX" versions could be done either as a part of this, later, or never)
   

So these would just change to `<ItemGroup Condition=" '$(TargetsDarwin)' ">`

Reply:
I went back and forth on this one and decided to keep them separate for now until we get the rest of the stack in and have a better understanding what can be shared and what can't.